app/models/work.rb

```diff
@@ -1220,7 +1220,9 @@ def bookmarkable_json
       pseud_ids: anonymous? || unrevealed? ? nil : pseud_ids,
       user_ids: anonymous? || unrevealed? ? nil : user_ids,
       bookmarkable_type: "Work",
-      bookmarkable_join: { name: "bookmarkable" }
+      bookmarkable_join: { name: "bookmarkable" },
+      public_word_count: word_count,
+      general_word_count: word_count
```

Comment:
For symmetry with the series tag PRs, I have added `public_word_count` and `general_word_count` fields. However, I wasn't sure if we should remove `word_count`, as perhaps it's used somewhere already? Alternatively, we could use `word_count` and `general_word_count`--let me know!